Libraries/Modal/Modal.js

```diff
@@ -223,17 +231,26 @@ class Modal extends React.Component<Props> {
       return null;
     }
 
+    const overlayColor =
+      this.props.overlayColor == null
+        ? null
+        : processColor(this.props.overlayColor);
+
     const containerStyles = {
       backgroundColor:
-        this.props.transparent === true ? 'transparent' : 'white',
+        this.props.transparent === true
+          ? 'transparent'
+          : this.props.overlayColor
```

Comment:
Should sort out the sketchy null check.

```suggestion
          : this.props.overlayColor != null
```